trade-finance-manager-api/src/v1/deals/routes.js

```diff
@@ -76,6 +77,10 @@ dealsAuthRouter
   .get(validation.dealIdValidation, handleExpressValidatorResult, dealController.getDeal)
   .put(validation.dealIdValidation, handleExpressValidatorResult, dealController.updateDeal);
 
+dealsAuthRouter
```

Comment:
Is this restricted to `PIM` only?

Reply:
It will be, we currently don't have any nice middleware validation to add in the tfm-api layer to check this. We've got a subtask on the ticket to create middleware on the tfm-api layer to do this

Reply:
Resolving this, I left the same comment - this will be added in another ticket/PR to keep PRs small 👍 @abhi-markan